packages/fluentui/react-icons-northstar/src/components/ComputerNoAudioIcon.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import { createSvgIcon } from '../utils/createSvgIcon';
+
+export const ComputerNoAudioIcon = createSvgIcon({
```

Comment:
We have `SpeakerMuteIcon`, should be this `ComputerAudioMuteIcon`?

![image](https://user-images.githubusercontent.com/14183168/87543571-42bff000-c6a5-11ea-9cf5-2b2d4408822b.png)